pkg/provisioner/helpers.go

```diff
@@ -236,6 +236,35 @@ func storageClassForObjectBucket(ob *v1alpha1.ObjectBucket, c kubernetes.Interfa
 	return class, nil
 }
 
+func addLabels(obj metav1.Object, newLabels map[string]string) {
```

Comment:
sorry I didn't look it up, but is this a pointer or by-value?

Reply:
`metav1.Object` is an interface. I did a quick test in the go playground to make sure for myself that this will work, and I think it works as expected.

https://play.golang.org/p/7EtZ-WiZSJT